test/OpenTelemetry.AutoInstrumentation.Tests/Configurations/EnvironmentInitializerTests.cs

```diff
@@ -69,7 +68,7 @@ public void Noop_WhenSettingIsNonOTel()
             });
             var actual = Environment.GetEnvironmentVariable(NonOtelVariableName);
 
-            actual.Should().BeNullOrEmpty();
+            Assert.True(string.IsNullOrEmpty(actual));
```

Comment:
I think it should be equal:

```suggestion
            Assert.Empty(actual);
```

Reply:
`Assert.Empty` will throw on `null` values, so I think the current code is a more accurate mapping.

Reply:
ah correct... messed up something while looking at the xunit src.

In that case `userMessage` would be good to add also.

Reply:
I reviewed Assert.True/False and filled part of the with additional messages.
Other looks fine to me, but I am open for suggestions.